packages/runtime-core/src/scheduler.ts

```diff
@@ -11,34 +11,33 @@ export function nextTick(fn?: () => void): Promise<void> {
 }
 
 export function queueJob(job: () => void) {
-  if (queue.indexOf(job) === -1) {
+  if (!queue.includes(job)) {
     queue.push(job)
     if (!isFlushing) {
       nextTick(flushJobs)
     }
   }
 }
 
-export function queuePostFlushCb(cb: Function | Function[]) {
-  if (Array.isArray(cb)) {
-    postFlushCbs.push.apply(postFlushCbs, cb)
-  } else {
-    postFlushCbs.push(cb)
+export function queuePostFlushCb(cbs: Function | Function[]) {
+  if (!Array.isArray(cbs)) {
+    cbs = [cbs]
   }
+
+  postFlushCbs.push(...cbs)
+
   if (!isFlushing) {
     nextTick(flushJobs)
   }
 }
 
-const dedupe = (cbs: Function[]): Function[] => Array.from(new Set(cbs))
+const dedupe = (cbs: Function[]): Function[] => [...new Set(cbs)]
 
 export function flushPostFlushCbs() {
   if (postFlushCbs.length) {
     const cbs = dedupe(postFlushCbs)
     postFlushCbs.length = 0
-    for (let i = 0; i < cbs.length; i++) {
```

Comment:
`for...of` is slower than a normal loop. We actively avoid using `for...of` in the codebase.

Reply:
@yyx990803 what's about `array.forEach`?

Reply:
It's a bit better but we only use it in non perf-critical places. Here a plain for loop is still preferred.

Reply:
Okay

Reply:
done!